### ISSUE TITLE
 Post \publication gives a bug with the upload_relation

### ISSUE THREAD

**Issue by cmaixen** [PubDig/pubdig#16]
Posting a publication fails by a certain error in the backend with upload_relation. There seems to be a problem with the user_id. It cannot retrieve the user_id from the request, although we include our cookie in the request. Including the cookie as authentications works for all the rest, so i suppose there has to be a problem in the backend.

```Test /publications
    POST
{ [Error: ER_BAD_NULL_ERROR: Column 'user_id' cannot be null]
  code: 'ER_BAD_NULL_ERROR',
  errno: 1048,
  sqlState: '23000',
  index: 0,
  isDuplicate: [Function] }
Caused by:INSERT INTO `%prefix%upload_relation` (user_id, publication_id, date_uploaded) VALUES (NULL, 1, NOW())
test
{ description: 'database failed' }
error: { description: 'database failed' }```

**Comment by cmaixen** [comment]
Authentication during the testing is solved by @VincentDS in #15 .

**Comment by cmaixen** [comment]
Fixed by login in in the beginning of the test. Reason of failure is an invalid cookie exported from previous tests.
